### ISSUE TITLE
feature request: have a way to show gene, intron, exon annotation tracks

### ISSUE THREAD

**Issue by timodonnell** [hammerlab/pileup.js#10]
Also, a way to jump to a gene by typing it in somewhere would be really useful.

My prompt for this is I am scrolling through TP53 in Cycledash / dalliance looking for any possible indication of variation, since our variant caller didn't find any variants there.

**Comment by timodonnell** [comment]
Ah, just realized that this is in fact possible in dalliance. Something similar to that would be useful.

**Comment by arahuja** [comment]
On this IGV also allows you to jump to a gene:
![image](https://cloud.githubusercontent.com/assets/455755/6378035/647efc04-bcf6-11e4-85e9-dd084db7fa43.png)

**Comment by jorvis** [comment]
Yes, please, this.  This is the reason I was searching for a library like this.
